packages/utils/src/dom/disposal.ts

```diff
@@ -96,8 +96,7 @@ export function cleanNode (node: Node) {
 
     // ... then its descendants, where applicable
     if (cleanableNodeTypesWithDescendants[node.nodeType]) {
-      // cleanNodesInList((node as CleanableNodeWithDescendants).getElementsByTagName("*"))
-      cleanNodesInList((node as CleanableNodeWithDescendants).childNodes)
+      cleanNodesInList((node as any).getElementsByTagName('*'));
```

Comment:
Why not use childNodes?  How are these two options here different?

Reply:
@brianmhunt the latter is recursive (grandchildren nodes). I changed it to childNodes under the assumption it'd be the same, but it broke the unit tests.

Reply:
Ah I see.  That’s unfortunate since childNodes is much faster .... but evidently incorrect 😀